docs/source/img/dvc_ex.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
```

Comment:
I left this file in case you wanted to change the plot. Otherwise it is not useful and can be removed.

Reply:
I like the figure, but it has a white background and the RTD scheme is light gray. Maybe we can make it transparent?